### PR TITLE
Workshop creation WMS ID 11462 

### PR DESCRIPTION
**Pull request by eliana-romero**
# Add here your WMS ID

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

**Comment by denisemyrick** [comment]
Hi Eliana please update branch before our review.

**Comment by eliana-romero** [comment]
Thank you Denise, I've just updated it, let me know if there is something else to do. Thank you so much

**Review by denisemyrick — CHANGES_REQUESTED** [review]

File: oci-analytics-fast-track-espanol/ft-create-dataset-es-lab-dos/ft-create-dataset-es-lab-dos.md
Comment (dannymgithub) [inline oci-analytics-fast-track-espanol/ft-create-dataset-es-lab-dos/ft-create-dataset-es-lab-dos.md:1]: @eliana-romero your png files should also be lowercased , example: ./images/lab2_t1_1.png

**Review by dannymgithub — COMMENTED** [review]
please review you png files, they are also case sensitive

**Review by anooshapilli — COMMENTED** [review]
@eliana-romero - Please make these images lowercase and rename them to be descriptive
image
image

Please use - or _ instead of spaces and move the file to files folder
image
image

**Comment by anooshapilli** [comment]
@eliana-romero - Just following up on the requested changes to the PR

**Comment by anooshapilli** [comment]
@eliana-romero - This PR is open for more than 5 days. Please make the requested changes and create a new PR. Closing the PR for now.

**Pull request closed without merging**